### ISSUE TITLE
BUG: device.send not working outside of setup()

### ISSUE THREAD

**Issue by pkerspe** [krygacz/arduino-eiscp#7]
While the example code works just fine for sending a command to the audio receiver, I do not have any success sending messages outside of the setup function.

The following code (run on ESP32 with version v0.2.2) will result in a single "MVLUP" command being sent to the receiver (from the setup function), but as soon as program is in the loop() function, all calls to device.send seem to be ignored/not processed anymore.

Here is a simple example program to reproduce:
```
#include <Arduino.h>
#include "WiFi.h"
#include "WiFiClient.h"
#include "eISCP.h"

const char *ssid = "givemeaname";
const char *password = "youwish";
const char *device_ip = "IP-OF-AUDIO-RECEIVER";
int device_port = 60128; 

WiFiClient client;
eISCP device(device_ip, device_port, &client);
void my_function(eISCP_Message message);

void connectToWiFi()
{
  Serial.print("\n\nConnecting to ");
  Serial.println(ssid);

  WiFi.mode(WIFI_STA);
  WiFi.begin(ssid, password);
  while (WiFi.status() != WL_CONNECTED)
  {
    delay(500);
    Serial.print(".");
  }
  Serial.println("\nConnected!");
}

void setup()
{
  Serial.begin(115200);
  connectToWiFi();
  device.send("MVLUP");
}

void loop()
{
  device.handle();
  device.send("MVLUP");
  delay(1000);
  device.send("MVLDOWN");
  delay(1000);
}
```

**Comment by krygacz** [comment]
Just published [v0.2.3](https://github.com/krygacz/arduino-eiscp/releases/tag/v0.2.3) which should fix this

**Comment by pkerspe** [comment]
I will test it, as soon as the new version is in the platformio repo, ususally takes 24h. Will send feedback tomorrow
